src/Insurance/Domain/Actions/RunPVerifyEligibilitySummaryAction.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Insurance\Domain\Actions;
+
+use Illuminate\Support\Facades\Log;
+use Lightit\Insurance\App\Exceptions\NotEligibleException;
+use Lightit\Insurance\Domain\DataTransferObjects\EligibilityCheckDto;
+use Lightit\Shared\Integrations\PVerify\DataTransferObjects\EligibilitySummaryRequestDTO;
+use Lightit\Shared\Integrations\PVerify\PVerifyConnector;
+use Lightit\Shared\Integrations\PVerify\Requests\EligibilitySummaryRequest;
+use Lightit\Shared\Integrations\PVerify\Requests\GetEligibilitySummaryResultsRequest;
+
+class RunPVerifyEligibilitySummaryAction
+{
+    public const MEDICARE_PAYER = 'Medicare Part A and Part B';
+    public const MEDICARE_PAYER_CODE = '00007';
+
+    public function __construct(
+        private readonly PVerifyConnector $connector,
+    ) {
+    }
+
+    public function execute(EligibilityCheckDto $eligibilityCheckDto, array $mbiLookupResponse): array
```

Comment:
mbiLookupResponse could just be the mbi string and not the whole response array, but if we wanted to keep the whole response we could create a dto, an array is a middle ground